packages/node_modules/@webex/widget-message/src/container.js

```diff
@@ -753,6 +755,7 @@ export class MessageWidget extends Component {
         <Dropzone
```

Comment:
Dropzone is from `react-drop-zone`, so no additional unit tests are needed for this use case.